pkgs/native_toolchain_c/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## 0.5.5-wip
 
-- Nothing yet.
+- Address analyzer info diagnostic about multi-line if requiring a block body.
```

Comment:
Did analyzer rules change?

Reply:
It's very strange indeed! It may be that a new Dart SDK was picked up on CI and that somehow makes this analyzer info diagnostic appear now - not sure. (The reason for the change was that CI showed me this as an error)